server/js/sockets.js

```diff
@@ -102,7 +102,19 @@ sockets.init = function socketInit(server) {
   } else {
     mqttClient = mqtt.connect('mqtt://localhost:1883', mqttOptions);
   }
-  
+  mqttClient.subscribe(DAS.start);
+  mqttClient.subscribe(DAS.stop);
+  mqttClient.subscribe(DAS.data);
+  mqttClient.subscribe(WirelessModule.all().module);
+  mqttClient.subscribe(BOOST.prev_trap_speed);
```

Comment:
@kdav108 Moved all these subscriptions down this file in https://github.com/monash-human-power/dashboard/pull/133, so now they're duplicated. Just need to move your new topic to where they now sit, and remove these lines